uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/filters/due-expiry.filter.js

```diff
@@ -1,93 +1,90 @@
+/* eslint-env amd */
+
 define([
   'tasks-assignments/filters/filters'
 ], function (filters) {
   'use strict';
 
-  filters.filter('filterByDue',['$filter', '$log', function($filter, $log){
-    $log.debug('Filter: filterByDue');
-
-    return function(inputArr, type, dateRange) {
+  filters.filter('filterByDueExpiry', ['$filter', '$log', function ($filter, $log) {
```

Comment:
The name `filterDueExpiry` is tightly coupled with field names. What if we need to add another field in future, then we would need to change the filter name to `filterDueExpiryXY`etc. Its better to name it something like `filterByDateField`.